clients/ethereumjs/ethereumjs.sh

```diff
@@ -46,7 +46,7 @@
 # Immediately abort the script on any error encountered
 set -e
 
-CLIENT_DIRECTORY=/ethereumjs-monorepo/packages/client
+CLIENT_DIRECTORY=/usr/app/packages/client
```

Comment:
If the path in this file was modified, it feels to me like the other two Docker files also need to be updated (Dockerfile.git, Dockerfile.local)

Reply:
I've made an update to the ethereumjs dockerfile to use the same client directory path name as before. Once [#3987](https://github.com/ethereumjs/ethereumjs-monorepo/pull/3987) is merged, this should be addressed.